tests/processors/test_renderers.py

```diff
@@ -275,6 +275,33 @@ def test_newline_in_value(self):
 
         assert r"with_newline=some\nvalue" == rv
 
+    @pytest.mark.parametrize(
+        ("raw", "escaped"),
+        [
+            # Slash by itself does not need to be escaped.
+            (r"a\slash", r"a\slash"),
+            # A quote requires quoting, and escaping the quote.
+            ('a"quote', r"a\"quote"),
```

Comment:
This one's not right: if there is a quote in the string, then the output needs to be between quotes, so:
```suggestion
            ('a"quote', r'"a\"quote"'),
```

That'll change the implementation, I guess.

Reply:
ugh thanks this stuff is so hard to enter